doc/source/methods/CFProto.ipynb

```diff
@@ -14,6 +14,21 @@
     "# Counterfactuals Guided by Prototypes"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Note\n",
+    "    \n",
+    "To enable support for Counterfactuals guided by Prototypes, you may need to run\n",
+    "\n",
+    "```bash\n",
+    "pip install alibi[tensorflow]\n",
+    "```\n",
+    "</div>"
```

Comment:
Same comment as above, can we make the box title be "Note" and the text part of the body?